src/java/main/org/apache/jute/Utils.java

```diff
@@ -190,19 +190,32 @@ static String fromCSVString(String s) throws IOException {
     }
     
     /**
+     * convert byte array to a string in hex format
      * 
-     * @param s 
-     * @return 
+     * @param barr
+     * @return
      */
-    static String toXMLBuffer(byte barr[]) {
```

Comment:
Hi @alburthoffman -

I have been trying to reproduce the issue that you described. A stack trace would be helpful here. Do you know exactly which test was failing for you (as I don't think the logs for any relevant jenkins job still exists)? I was able to get the whole test suite to pass with `DatatypeConverter.printHexBinary`. 

Although, if I do not add null check I can get `testJuteToString` to fail. 

```
java.lang.NullPointerException
	at javax.xml.bind.DatatypeConverterImpl.printHexBinary(DatatypeConverterImpl.java:479)
	at javax.xml.bind.DatatypeConverter.printHexBinary(DatatypeConverter.java:626)
	at org.apache.jute.Utils.toHexString(Utils.java:203)
	at org.apache.jute.Utils.toCSVBuffer(Utils.java:242)
	at org.apache.jute.CsvOutputArchive.writeBuffer(CsvOutputArchive.java:100)
	at org.apache.zookeeper.proto.SetDataRequest.toString(SetDataRequest.java:77)
	at org.apache.zookeeper.server.ToStringTest.testJuteToString(ToStringTest.java:36)
```

I was able to remedy this with a null check before the javax call. 

```
    private static String toHexString(byte barr[]) {
        if (barr == null) {
            return "";
        }
        return javax.xml.bind.DatatypeConverter.printHexBinary(barr);
    }
```

Thanks for your patience on this.

Abe